src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1585,35 +1584,46 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         }
     }
 
+
     handleNameOnCardFieldChange() {
+        this.setCardHolderName();
```

Comment:
@bdvorachek We need to be able to read the card holder name `onChange` of the Donor Type field of the form and when the `firstName` and `lastName` fields are not on the form, it seems the dataImport record only gets constructed onLookUpRecordSelect. Depending on the order in which you populate data in the form, the names do not get fabricated. I think we could queue up the selected `recordId` for retrieval using the `queueSelectedRecordForRetrieval` function